netfields/address_families.py

```diff
@@ -0,0 +1,46 @@
+from ipaddress import ip_interface, IPv4Interface, IPv6Interface, ip_network, IPv4Network, IPv6Network, ip_address, \
+    IPv4Address, IPv6Address
+
+BOTH_FAMILIES = None
+IPV4_FAMILY = 4
```

Comment:
If this is going to be namespaced into a module like this, it might be better to call these `IPV4` and `IPV6`. Also instead of `BOTH` could use `UNSPECIFIED` which is probably more correct as an address can't be in 2 families. :-)